lib/ast.js

```diff
@@ -646,11 +644,10 @@ var AST_Definitions = DEFNODE("Definitions", "definitions", {
         definitions: "[AST_VarDef*] array of variable definitions"
     },
     _walk: function(visitor) {
-        return visitor._visit(this, function(){
-            var definitions = this.definitions;
-            for (var i = 0, len = definitions.length; i < len; i++) {
-                definitions[i]._walk(visitor);
-            }
+        return visitor._visit(this, function() {
+            this.definitions.forEach(function(defn) {
+                defn._walk(visitor);
+            });
```

Comment:
@fabiosantoscode I was hoping these loop code changes wouldn't be ported. They slow down execution in my tests.